valmlord/gulpify
# feat: add babel and webpack to gulp boilerplate

## Description
This pull request enhances the Gulp boilerplate setup by integrating Babel and Webpack. It also establishes a basic folder structure for JavaScript files.

## Changes
### Babel and Webpack Integration
- Added necessary dependencies (`@babel/core`, `@babel/preset-env`, `webpack`, etc.) to `package.json`.
- Updated the `gulpfile.js` to include tasks for transpiling JavaScript using Babel and bundling with Webpack.

### Base Folder Structure for JS Files
- Created a `src/js` directory to store JavaScript source files.
- Added an initial sample JavaScript file (`app/js/main.js`) to demonstrate the structure.

## How to Test
1. Execute the relevant Gulp tasks (e.g., `gulp buildJS`) to transpile and bundle JavaScript.
2. Verify that the output files are correctly generated in the `build/js` directory.

## Additional Notes
- This update aims to improve the build process by enabling modern JavaScript features and bundling.
